devices/tuya.js

```diff
@@ -1698,22 +1697,14 @@ module.exports = [
         exposes: [e.battery(), e.temperature(), e.humidity(), e.battery_voltage()],
     },
     {
-        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_yjjdcqsq']),
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_yjjdcqsq', '_TZE200_9yapgbuv']),
         model: 'ZTH01',
         vendor: 'TuYa',
         description: 'Temperature and humidity sensor',
-        fromZigbee: [tuya.fz.datapoints, tuya.fz.gateway_connection_status],
-        toZigbee: [tuya.tz.datapoints],
+        fromZigbee: [fz.tuya_temperature_humidity_sensor],
+        toZigbee: [],
         configure: tuya.configureMagicPacket,
         exposes: [e.temperature(), e.humidity(), tuya.exposes.batteryState(), e.battery_low()],
-        meta: {
```

Comment:
Please undo these changes (`fromZigbee`/`toZigbee` should stay as is).

Reply:
Thanks for the review! :)

I have reverted the removal of meta.
But I don't quite understand why it is needed?
I thought the converter `fz.tuya_temperature_humidity_sensor` mapped the datapoints and did any computation needed (eg. `divideBy10`).

But this didn't quite explain why, only how to use it:
https://www.zigbee2mqtt.io/advanced/support-new-devices/02_support_new_tuya_devices.html#_4-mapping-the-datapoints

Reply:
It does indeed, but the separate converter for each device is the old way of integrating TuYa devices. Now just  `tuya.tz.datapoints/`tuya.fz.datapoints` should be used. Do you experience any problems when doing that?